src/test/kotlin/minesweeper/domain/BoardFieldsTest.kt

```diff
@@ -56,4 +57,97 @@ class BoardFieldsTest : DescribeSpec({
             boardFields.mineCount() shouldBe 3
         }
     }
+
+    describe("open") {
+        context("필드들에 해당하는 좌표를 입력받아") {
+            it("해당하는 필드를 열 수 있다.") {
+                val boardFields = boardFields()
+                boardFields.open(Coordinate(0, 0))
+
+                val boardField = boardFields.boardFields.find { it.coordinate == Coordinate(0, 0) }!!
```

Comment:
```suggestion
                val boardField = boardFields.boardFields.first { it.coordinate == Coordinate(0, 0) }
```
non-null assertion 기호를 `!!`이라는 불편한 기호를 채택한 이유는 최대한 사용을 자제하기 위해서라고 하네요. 코틀린의 많은 기본 API는 null을 리턴하는 api와 non-null을 리턴하는 api를 모두 제공합니다. 한번씩 찾아보시면 좋을 것 같아요!